test/unit/data/events/scheme/order-grouped-events-by-scheme.test.js

```diff
@@ -0,0 +1,46 @@
+
+const { orderGroupedEventsByScheme } = require('../../../../../app/data/events/scheme-id/order-grouped-events-by-scheme')
+const { SFI, SFI_PILOT, LUMP_SUMS, VET_VISITS, CS, BPS, FDMR } = require('../../../../../app/constants/schemes')
+
+let groupedEvents
+let groupedEvent
+
+let sfiEvent
+let sfiPilotEvent
+let lumpSumsEvent
+let vetEvent
+let csEvent
+let bpsEvent
+let fdmrEvent
+
+describe('order grouped events', () => {
+  beforeEach(() => {
+    groupedEvent = JSON.parse(JSON.stringify(require('../../../../mocks/events/grouped-event')))
+    sfiEvent = { ...groupedEvent, schemeId: SFI }
+    sfiPilotEvent = { ...groupedEvent, schemeId: SFI_PILOT }
+    lumpSumsEvent = { ...groupedEvent, schemeId: LUMP_SUMS }
+    vetEvent = { ...groupedEvent, schemeId: VET_VISITS }
+    csEvent = { ...groupedEvent, schemeId: CS }
+    bpsEvent = { ...groupedEvent, schemeId: BPS }
+    fdmrEvent = { ...groupedEvent, schemeId: FDMR }
+    groupedEvents = [bpsEvent, vetEvent, sfiEvent, sfiPilotEvent, lumpSumsEvent, csEvent, fdmrEvent]
+  })
+
+  test('should order grouped events into sequence by schemeId when all schemes present', () => {
```

Comment:
What does 'by sequence' mean here?

Reply:
Would schemeId be in ascending order then?

Reply:
yep, have added that to test name